### PR TITLE
Tones down fetish fuel from the 'Waifu' drink

### PR DESCRIPTION
**Pull request by Gringonius**
Really.
[tested]

### CODE REVIEW

**Comment by jknpj** [comment]
Are you implying that traps are better

**Comment by yclatious** [comment]
Are you implying they are not jk?

**Comment by beesil** [comment]
nothing wrong with magical girl(male)

**Comment by Dacendeth** [comment]
gender change surgery is a really simple surgery if forced gender changes bothers you

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -5397,14 +5397,12 @@
...
 		return 1
-	if(M.gender == MALE)
-		M.setGender(FEMALE)
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
 		if(!M.is_wearing_item(/obj/item/clothing/under/schoolgirl))
 			var/turf/T = get_turf(H)
 			T.turf_animation('icons/effects/96x96.dmi',"beamin",-32,0,MOB_LAYER+1,'sound/effects/rejuvinate.ogg',anim_plane = MOB_PLANE)
-			H.visible_message("<span class='warning'>[H] dons her magical girl outfit in a burst of light!</span>")
+			H.visible_message("<span class='warning'>[H] dons their magical girl outfit in a burst of light!</span>")
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm:5405]: It should be `\his` (or `\her`)
It should be that already, too

**Comment by FatCat1978** [comment]
cross dressing is my fetish please merge asap

**Comment by SonixApache** [comment]
[missing the point of the drink]

**Comment by Intigracy** [comment]
traps are waifus too

**Comment by strawberrytan** [comment]
nothing wrong with waifu

**Comment by carshalash** [comment]
Local degenerate forces trap fetish on pure space station players.

**Comment by Exxion** [comment]
The emojiocracy has spoken pretty clearly

**Pull request closed without merging**